### PR TITLE
Some changes in the header so Godot3 compiles again on Windows.

### PR DESCRIPTION
**Pull request by BastiaanOlij**
Just a few minor changes in the math libraries so the right header files are included and everything compiles again...

### CODE REVIEW

File: core/math/math_funcs.h [hunk 1]
```diff
@@ -33,10 +33,9 @@
 #include "math_defs.h"
 #include "pcg.h"
 
-#ifndef NO_MATH_H
```
Comment (bojidar-bg) [inline core/math/math_funcs.h]: Why kill the ifdef?

Comment (akien-mga) [inline core/math/math_funcs.h]: Because it's useless, nothing ever defines it.

Comment (bojidar-bg) [inline core/math/math_funcs.h]: Ah, all good then :smiley:

Comment (BastiaanOlij) [inline core/math/math_funcs.h]: @bojidar-bg cause @akien-mga made me do it ;)

**Pull request merged**